spec/workflow/states/fail_spec.rb

```diff
@@ -1,8 +1,13 @@
 RSpec.describe Floe::Workflow::States::Fail do
   let(:workflow) { Floe::Workflow.load(GEM_ROOT.join("examples/workflow.asl")) }
   let(:state)    { workflow.states_by_name["FailState"] }
+  let(:inputs)   { {} }
 
   it "#end?" do
     expect(state.end?).to be true
   end
+
+  it "has a next of nil" do
+    expect(state.run!(inputs).first).to be nil
```

Comment:
I would have expected a `.next` call here - not sure what .first is.

Reply:
`run` currently returns `[next, output]`, so it is the next part. I will make more clear